Dockerfile

```
@@ -1,75 +1,59 @@
 ARG TARGET=server
-
-# Can be used in case a proxy is necessary
 ARG GOPROXY
 
+##### Temporal builder #####
 FROM temporalio/base-builder:1.0.0 AS temporal-builder
-
-# Making sure that dependency is not touched
-ENV GOFLAGS="-mod=readonly"
-
 WORKDIR /temporal
+ENV CGO_ENABLED 0
 
-# Copy go mod dependencies first and build docker cache
-COPY go.mod ./
-COPY go.sum ./
+# Copy go.mod first to build docker layer with go dependencies (to improve rebuild time).
+COPY go.mod .
+COPY go.sum .
```

```suggestion
COPY go.mod go.sum .
```
Please use single layer